vipps-ecom-api.md

```diff
@@ -1566,28 +1567,30 @@ Example of the userInfo flow:
 
 See the API specification for the formats and other details for each scope.
 
-**Please note:** If the e-mail address that is delivered has the flag `email_verified : false`
+**Please note:** If the e-mail address that is delivered has the flag `email_verified : false`,
 this address should not be used to link the user to an existing account without
 further authentication. Such authentication could be to prompt the user to
-login to the original account or confirm the account linking by having a
+log in to the original account or to confirm the account linking by providing a
 confirmation link sent to the email address.
 
-### Userinfo call by call guide
+### Userinfo call-by-call guide
 
 Scenario: You want to complete a payment and get the name and phone number of
-a customer.
+a customer. Details about each step are described in the sections below.
 
 1. Retrieve the access token:
-   [`POST:/accesstoken/get`](https://vippsas.github.io/vipps-recurring-api/#/Access%20Controller/getAccessToken).
-2. Add scope to the transaction object and include the scope you wish to get
-   access to (valid scope) before calling
-   [`POST:/ecomm/v2/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST)
-3. The user consents to the information sharing and perform the payment in Vipps.
-4. Retrieve the `sub` by calling
-   [`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/getPaymentDetailsUsingGET)
-5. Using the sub from step 4, call
+   [`POST:/accesstoken/get`](https://vippsas.github.io/vipps-ecom-api/#/Authorization%20Service/fetchAuthorizationTokenUsingPost).
+2. Add `scope` to the
+   [`POST:/ecomm/v2/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST) call and include the scopes you need access to (e.g., "name address email phoneNumber birthDate"). Separate scopes with spaces.
+3. The user consents to sharing the information and performs the payment in Vipps.
+4. You retrieve the user identifier, `sub`, by calling
+   [`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/getPaymentDetailsUsingGET).
+5. Retrieve a login token via the Vipps Login API:
```

Comment:
Added a new step here. You have to get this login token for the next call to work.